### DISCUSSION TITLE
Closed Source and Open Source Tools Comparison

### DISCUSSION THREAD

**Discussion by KenNL42** [albertkun/21S-ASIAAM-191A#107] [Past Thinking Caps]
In terms of software, closed source software outperforms open source in terms of data usability and service. The information from closed source tend to target to a specific audience, which can limit scope and help developers create a program with ease. Close source also generally provides more helps with its data when it comes to debugging service, which you cannot always rely open source tool to be fixed as soon as possible. On the other hand, closed source can cost lots of money for developers to use. If a project scale is small, then closed source tools may be too excessive to use.
When it comes to civil society, the above points also applies to civil society tools. One example of closed source tool is [Google Maps API](https://cloud.google.com/maps-platform/pricing/sheet). Google advertises its product that there are 25 million updates daily and cover 99% of the world. It also offers community support from experts and employees. However, Google Maps API can charge you after using above a free $200 monthly credit. This can be enough for small project, but a big one can cost too much. Thus, closed source can be harmful in term of cost, while open source can be harmful in term of usability and service.

**Comment by avaitt** [comment]
I like the fact that you mention closed source tends to target niche audiences. I never realized that by having a consumer pay for a service you narrow the intended audience of that service. Of course, closed source by the same token can act as a repellent and prevent small-scale developers from growing out their products. I think the conversation about privacy we need to have about open-source software and data collection. Clearly, companies will not be letting go of our data anytime soon. We need to ask the question: At what point do we draw the line with our data? When does the data vs. service fee tradeoff crumble?

**Comment by nnhimichelle** [comment]
Thank you for this explanation, I enjoyed reading it! To echo on what Adhvaith stated above, it is great that you mentioned how closed source software tends to target specific audiences, as I did not think of it in that scope and in more of just the perspective where they could be unintentionally excluding other audiences. I also definitely agree on what you stated about troubleshooting and project scale services, as it definitely can being up a large conflict when deciding which tools would be best for projects and whether the cost of such services would be worth it.
